content/engine/install/centos.md

```diff
@@ -39,14 +39,14 @@ Uninstall any such older versions before attempting to install a new version,
 along with associated dependencies.
 
 ```console
-$ sudo yum remove docker \
-                  docker-client \
-                  docker-client-latest \
-                  docker-common \
-                  docker-latest \
-                  docker-latest-logrotate \
-                  docker-logrotate \
-                  docker-engine
+$ sudo yum remove docker-ce \
```

Comment:
I don't think this is correct; the intent here was to uninstall the package named `docker` (if installed), which is an older name of a package that shipped docker. The intent is not to uninstall a  previous install of the `docker-ce` package (which is the current name of the package)